Libraries/Text/RCTShadowText.m

```diff
@@ -326,7 +326,9 @@ - (NSAttributedString *)_attributedStringWithFontFamily:(NSString *)fontFamily
   [self _addAttribute:NSFontAttributeName withValue:font toAttributedString:attributedString];
   [self _addAttribute:NSKernAttributeName withValue:letterSpacing toAttributedString:attributedString];
   [self _addAttribute:RCTReactTagAttributeName withValue:self.reactTag toAttributedString:attributedString];
-  [self _setParagraphStyleOnAttributedString:attributedString heightOfTallestSubview:heightOfTallestSubview];
+  [self _setParagraphStyleOnAttributedString:attributedString
+                                    fontSize:[font pointSize]
```

Comment:
Style: Use property notation (`.`) when the iOS docs document members like pointSize as properties.

<img width="215" alt="fontsize" src="https://cloud.githubusercontent.com/assets/379606/17413244/4c35a566-5a35-11e6-8f2d-3b9dfa9be9d7.png">